main.go

```diff
@@ -132,14 +132,23 @@ func setPalette() {
 		return
 	}
 	vals := strings.Split(v, ",")
-	if len(vals) != 2 {
+	if len(vals) > 3 {
```

Comment:
We could avoid erroring on incorrect use of the env variable and make this a bit more compact with something like this:

```
    vals := strings.Split(v, ",")
    states := []*color.Color{fail, success, skip}
    for i := range vals {
        if c, ok := colors[vals[0]]; ok {
            states[i] = color.New(c)
        }
    }
```

Reply:
Let me test that out

Reply:
I believe I got it working, with your suggestion.